src/components/domain/project/projectButton/ProjectRegisterButton.tsx

```diff
@@ -29,11 +33,12 @@ export default function ProjectRegisterButton({ className }: ProjectRegisterButt
   };
 
   return (
-    <div className={cn(className)}>
-      <Button onClick={handleOpenProject} variant={'gradient'} className="h-[60px] w-[250px]">
-        <ClipboardEdit className="mr-2 h-5 w-5" />
-        <p className="body6">내 프로젝트 등록하기</p>
-      </Button>
-    </div>
+    <Button
+      onClick={handleOpenProject}
+      variant={'gradient'}
+      className={(cn('h-[60px] w-[250px]'), className)}>
```

Comment:
```suggestion
      className={cn('h-[60px] w-[250px]', className)}>
```
여기 괄호가 잘못된 것 같습니다. 지금은 cn('h-[60px] w-[250px]')여기가 동작하지 않고 className만 반환되는거 아닌가요..?

Reply:
✅ 7022c127b50ebc7c930190610e2bf69f3592ca7e
오....맞습니다 ... 수정했습니다!